pkg/ebpf/lockcontention_nolinuxbpf.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: consider renaming the file to lockcontention_noop.go to follow "unwritten" pattern